### DISCUSSION TITLE
Make multiplayer spectator background follow user setting for main menu background

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#24510] [Ideas] [answered]
I remember this working at some point, though every time spectator started it would still show a seasonal background so you had to go Beatmap bg -> other option -> beatmap bg to make it work everytime. Now it looks like it never shows the beatmap background.

This makes sense to me, but i can understand if this behaviour being tied to a setting under "Main Menu" isn't ideal.

https://github.com/ppy/osu/assets/53081750/8e680840-f384-4909-9a26-19cb155c16cb

(I will redo with logs when i wake up)

**Comment by peppy** [comment, marked as answer]
set seasonal to off. it overrides your choice here otherwise.
